pkg/util/pointer/pointer.go

```diff
@@ -52,6 +52,12 @@ func Int32Ptr(i int32) *int32 {
 	return &o
 }
 
+// Int64Ptr returns a pointer to an int64
+func Int64Ptr(i int64) *int64 {
+	o := i
+	return &o
```

Comment:
Why not simplify it to `return &i` ?

Reply:
Ah, probably, for consistency with `Int32Ptr()`? Ok.

Reply:
Yeah, this was just for consistency, but I can simplify all of the functions, it if you'd like.

Reply:
Updated w/ simplified functions.